source/_integrations/lifx.markdown

```diff
@@ -114,6 +116,53 @@ Run an effect that does nothing, thereby stopping any other effect that might be
 | ---------------------- | ----------- |
 | `entity_id` | String or list of strings that point at `entity_id`s of lights. Use `entity_id: all` to target all.
 
+## Buttons
+
+The LIFX button platform creates two buttons for each LIFX device:
+
+| Entity ID                      | Default Status |
+| ------------------------------ | -------------- |
+| `button.{bulb_label}_identify` | Enabled        |
+| `button.{bulb_label}_restart`  | Disabled       |
+
+### Identify Button
+
+The Identify button will flash the bulb three times at maximum brightness then return the bulb to the state it was in prior. Successful identification requires the bulb to be powered on and already configured in Home Assistant.
+
+### Identifying via Automation
+
+The following snippet demonstrates how to programmatically press the Identify button in a script or automation:
+
+```yaml
+identify_lifx_bulb:
+  alias: "Identify LIFX bulb"
+  sequence:
+    - service: button.press
+      target:
+        entity_id: button.kitchen_light_identify
+```
```

Comment:
I don't think we have to describe how button entities work, that documentation is more fitting in the button entity documentation.

We do exempt from those behaviors, in case there is very special case, but these are just regular buttons.